TESTS/psa/crypto_init/COMPONENT_SPE/psa_setup.c

```diff
@@ -1,83 +0,0 @@
-/* Copyright (c) 2017-2018 ARM Limited
- *
- * SPDX-License-Identifier: Apache-2.0
```

Comment:
what is the reason for removing this file as part of this PR?

Reply:
when the tool scans the psa test folder it should not generate a psa_setup.c file for tests that do not declare new partition

so this file is not needed anymore